docs/getting-started.md

```diff
@@ -110,6 +110,44 @@ You can now continue using your UP address within the dApp:
 const myUPAddress = myContracts.ERC725Account.address;
 ```
 
+## Options
+
+When instantiating LSPFactory options can be passed to specify parameters such as `chainId` and `ipfsGateway`.
```

Comment:
Suggested change:

_"The following parameters are available when creating an instance of the LSPFactory"_